Neos.SiteKickstarter/Resources/Private/AfxGenerator/Fusion/Document/AbstractPage.fusion

```diff
@@ -0,0 +1,58 @@
+prototype({packageKey}:Document.AbstractPage) < prototype(Neos.Neos:Page) {
+  head {
+    headTags = Neos.Fusion:Component {
+      renderer = afx`
+        <meta http-equiv="x-ua-compatible" content="ie=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      `
+    }
+
+    titleTag = Neos.Fusion:Tag {
+      tagName = 'title'
+      content = ${q(node).property('title')}
+    }
+
+    # Add stylesheets to your page
+    stylesheets.site = Neos.Fusion:Tag {
+      tagName = 'link'
+      attributes.rel = 'stylesheet'
+      attributes.href = Neos.Fusion:ResourceUri {
+        # Change to your file path
+        path = 'resource://{packageKey}/Public/site.css'
+      }
+    }
+
+    # Add header javascripts to your page
+    javascripts.site = Neos.Fusion:Tag {
+      tagName = 'script'
+      attributes.type = 'text/javascript'
+      attributes.src = Neos.Fusion:ResourceUri {
+        # Change to your file path
+        path = 'resource://{packageKey}/Public/site.js'
+      }
+    }
```

Comment:
```suggestion
    javascripts.site = afx`
        <link type="text/javascript" defer href={StaticResource.uri('{packageKey}', 'Public/site.js')} />
    `
```
We could also use here the new Eel helper. And I suggest also to add the defer attribute…